src/app/books/books-grid/book-card/book-card.component.html

```diff
@@ -5,48 +5,11 @@
   <div class="card-block">
     <h4 class="card-title">{{ book.id }} {{ book.title }}</h4>
     <p class="card-text">{{ book.author }}</p>
-    <button (click)="edit(content)" type="button" class="btn btn-primary">
+    <a href="#" class="btn btn-primary">
```

Comment:
Not href's in Angular code.

I rather `button` component than `a` component. More descriptive and Bootstrap compliant code.